### ISSUE TITLE
how to use create_macos_app_bundle.sh

### ISSUE THREAD

**Issue by Trixter69** [keithgg/puddletag#346]
Hey all,
I've been trying to create a Puddletag.app for mac but have NO idea how to use the create_macos_app_bundle.sh file. I keep getting a empty.app/Contents/Info.plist: No such file or directory error.

A little help would be appriciated.

**Comment by gesslerpd** [comment]
Hey,
I don't really know too much about this project but I looked into the script and found these lines setting up the command line arguments: https://github.com/keithgg/puddletag/blob/master/source/create_macos_app_bundle.sh#L54

`-n or --name` - To specify the app name
`-i or --icon` - To specify the app icon to use
`-s or --script` - To specify the startup script to use

So for example:
```shell
./create_macos_app_bundle.sh -n puddletag -i ./puddletag.png
```
Not really sure what to put for the startup script argument but I hope this helps you get started.

**Comment by Trixter69** [comment]
Thx, Figured it out, the script I had to use was the one under fink not the one in the GIT pull.

I used Fink to install puddletag cause it installs with QT4 dependencies, its a semi-closed environ so when I tried to use the puddletag from GIT pull it defaults fo the main system Python install which doesn't have QT4 so crashed.

Heres the command I ran that did work:
`sudo ./create_macos_app_bundle.sh -n /Applications/PuddleTAG -i ./puddletag.png -s /sw/bin/puddletag`

for anyone looking to install this on OS X Sierra I'd suggest installing Fink using the Fink Install tool:
`https://github.com/fink/scripts/blob/master/srcinstaller/Install Fink.tool`
Copy and paste the code into your favorite text editor, Save it as 'fink_installer.sh' on the desktop
open terminal and enter:
```
cd ~/Desktop/
./fink_installer.sh
```
hit opt+cmd+r to restart terminal when fink is finished installing then run the following commands:

`fink install puddletag` wait to finish installing then replace everything in '/sw/lib/python2.7/site-packages/puddlestuff/' with the new puddlestuff folder from the master release zip.

finally if you want an app Applications folder run `sudo ./create_macos_app_bundle.sh -n /Applications/PuddleTAG -i ./puddletag.png -s /sw/bin/puddletag` from the source folder.

Congratulations puddletag is installed and updated and can be run
